src/easyopt.c

```diff
@@ -1040,7 +1068,7 @@ do_curl_setopt(CurlObject *self, PyObject *args)
     /* early checks of option value */
     if (option <= 0)
         goto error;
-    if (option >= (int)CURLOPTTYPE_OFF_T + OPTIONS_SIZE)
+    if (option >= (int)CURLOPTTYPE_BLOB + OPTIONS_SIZE)
```

Comment:
`CURL_OPTTYPE_BLOB` was also added only in 7.71.0, so this line needs a conditional too

Reply:
Fixed